tests/containers/host_configuration.pm

```diff
@@ -50,7 +50,7 @@ sub run {
         } elsif ($host_distri eq 'centos') {
             assert_script_run("dhclient -v");
             script_retry("yum update -q -y --nobest", timeout => $update_timeout);
-        } elsif ($host_distri eq 'rhel') {
+        } elsif ($host_distri eq 'rhel' && $version ne '7') {
```

Comment:
Please add a comment that states that this is only a temporary solution due to bsc#